File: native/mixdog-graph/tests/fixtures/calls/python/sample.py
# hidden()
quoted = "hidden()"

def inner():
    return None

def leaf(x):
    return x

def nest(x):
    return x

def helper():
    return None

def seed():
    return None

def plain():
    return None

def deco():
    return lambda fn: fn

@deco()
def run(a):
    inner()
    nest(leaf(1))
    a.b().c()

class Widget:
    def ping(self):
        return None

    def act(self):
        helper()
        self.ping()

plain()
mark = "μ"; seed()
Widget()

File: native/mixdog-graph/tests/fixtures/calls/php/sample.php
<?php
// hidden()
$quoted = "hidden()";

function inner() {}
function leaf($x) { return $x; }
function nest($x) {}
function helper() {}
function seed() {}
function plain() {}

plain();

function run($a) {
  inner();
  nest(leaf(1));
  $a->b()->c();
}

class Widget {
  function ping() {}
  function act() {
    helper();
    $this->ping();
  }
}

$mark = "μ"; seed();
new Widget();
